### ISSUE TITLE
Prevent the plugin from opening tabs in chat.openai.com (or delete them)

### ISSUE THREAD

**Issue by TheWonderfulTartiflette** [wong2/chat-gpt-google-extension#160]
When using this plugin and going to the original ChatGPT page after a while, there are a lot of "New chat(s)" opened, always with the Google query. It would be REALLY nice for it to prevent opening chats like this, or delete them after use
![image](https://user-images.githubusercontent.com/44057002/210255300-5b8512bd-255d-4dea-85f1-1e822b824b91.png)

**Comment by wong2** [comment]
The extension does automatically delete those chats created by search. I doubt the deletion failed in some cases. I'll look into it.

**Comment by TheWonderfulTartiflette** [comment]
At least when I try to regenerate the responses in the screenshot, I get "Conversation not found" error

**Comment by wong2** [comment]
Does that mean the conversation was deleted, but ChatGPT website still shows them in history list?

**Comment by TheWonderfulTartiflette** [comment]
I don't think so, because actually (and sorry for the misinformation before) the "Conversation not found" appears only rarely, and most of the time it's normal conversation that is simply named "New chat"

**Comment by wong2** [comment]
🆗

**Comment by alexander-turner** [comment]
I'm also getting this issue.

**Comment by vgmda** [comment]
Same issue here

**Comment by arthurborgesdev** [comment]
Same issue. The button to delete conversations in the config is greyed out:

<img width="518" alt="image" src="https://user-images.githubusercontent.com/38326017/215909733-6d2e1bf0-72ef-446d-a164-9ca41b15ae2a.png">

**Comment by wong2** [comment]
For anyone facing this issue, can you help me debug it?

**Comment by arthurborgesdev** [comment]
> For anyone facing this issue, can you help me debug it?

Perhaps I'm being naive but removing the `disabled` here might solve the issue: 

https://github.com/wong2/chatgpt-google-extension/blob/eb6702c29df85ff1d8b73dbea19c93411f1e13d0/src/options/App.tsx#L141

**Comment by wong2** [comment]
@arthurborgesdev No of course not.

**Comment by bigplayer-ai** [comment]
I still experience the same issue, I want to delete the conversations from chatgpt website.

**Comment by bigplayer-ai** [comment]
> For anyone facing this issue, can you help me debug it?

I can help you with that.

**Comment by Hunter-Bobeck** [comment]
Auto-delete switch in extension settings is currently grayed out for me.

**Comment by liuzhenyu-yyy** [comment]
> Auto-delete switch in extension settings is currently grayed out for me.

Same issue here. Could not turn on "Auto delete conversations generated by search" in Edge.

**Comment by AnqiGuan** [comment]
> Auto-delete switch in extension settings is currently grayed out for me.↳

Same issue here. I remember a few days ago it did work, but now I'm not allowed to toggle the "Automatically delete conversations generated by search" button.

**Comment by FauconSpartiate** [comment]
![image](https://user-images.githubusercontent.com/49079695/223428114-37e487de-8022-4978-86c4-ba1a6090004e.png)
Same here, the conversations are spammed with "New Chat" that never get deleted.
@wong2 any ETA or idea on how this could be fixed?

**Comment by spotandjake** [comment]
Im having this issue too, and do not see the switch to toggle deletion in the config.
